packages/table/test/columnHeaderCellTests.tsx

```diff
@@ -143,22 +143,18 @@ describe("<ColumnHeaderCell>", () => {
     });
 
     describe("Reorder handle", () => {
-        const REORDER_HANDLE_CLASS = "reorder-handle";
+        const REORDER_HANDLE_CLASS = "bp-table-reorder-handle-target";
```

Comment:
It's not a test-only class.

...I should use `Classes.TABLE_REORDER_HANDLE_TARGET`.